_posts/2018-09-06-fullstack-typed-api-with-typescript.md

```diff
@@ -0,0 +1,427 @@
+---
+layout: post
+title: Fullstack shared typed API with TypeScript
+author: jgke
+excerpt: >
+  TypeScript makes it possible to share type information between frontend and
+  backend. Here's how to use it to ensure consistent API calls between the
+  frontend and backend.
+tags:
+ - TypeScript
+ - Full stack
+---
+
+Did you get yet another error due to the JavaScript frontend slipping an
+unnoticed ``undefined`` to a request or passing a string instead of a number,
```

Comment:
It doesn't seem to make any difference, but I think a single backtick is enough.